src/NLog.Web.AspNetCore/NLogRequestPostedBodyMiddlewareConfiguration.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+
+namespace NLog.Web
+{
+    /// <summary>
+    /// Contains the configuration for the NLogRequestPostedBodyMiddleware
+    /// </summary>
+    public class NLogRequestPostedBodyMiddlewareConfiguration
```

Comment:
Can it be renamed to NLogRequestPostedBodyMiddlewareOptions` ?

Reply:
Complete